src/app/[locale]/auth/createAccount.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+import Button from "@/app/daisyui/button";
+import { ErrorInfo, FormEvent, useState } from "react";
+import { FormInput } from "../(components)/formInput";
+import { IAccountRegister } from "@/gql/sdk";
+import { gqlClient } from "@/gql";
+
+export const CreateAccount = () => {
+  const [firstName, setFirstName] = useState<string>("");
+  const [lastName, setLastName] = useState<string>("");
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+  const [confirmPassword, setConfirmPassword] = useState<string>("");
+
+  return (
+    <div className="h-auto bg-slate-200 w-full flex flex-col items-center p-8">
```

Comment:
Update colors to use theme colors for back and foreground, use v2 mock styling (which puts a diff. color background behind "Create Account"